### ISSUE TITLE
Add info about maintenance status.

### ISSUE THREAD

**Issue by ashthespy** [volumio/Volumio2#2182]
Disable PRs/Issues and point people towards https://github.com/volumio/volumio3-backend?

**Comment by volumio** [comment]
@ashthespy yeah, I totally agree

**Comment by ashthespy** [comment]
What is to be done with all the open issues? A lot of them are still valid..

**Comment by volumio** [comment]
Yeah but it will take days to get trough them, I would close them IMHO and just open the most urgent onese on volumio3

**Comment by ashthespy** [comment]
Are you sure? There are a lot of issues (and fixes/hints/ideas) here that would then be lost. 
I don't think github lets you search issues in two repos easily..

**Comment by volumio** [comment]
I would prefer to keep them... but I don't have the time to go trough them, nor the support team..
If you can help by filtering the ones that are more relevant and port them, that would be appreciated

**Comment by ashthespy** [comment]
>I would prefer to keep them... but I don't have the time to go trough them, nor the support team..

Then it's a shame that we don't continue to use this repo! Especially considering most of the buster/v3 stuff was done already in this repos https://github.com/volumio/Volumio2/compare/master...buster/master
The codebase alone hasn't changed enough to warrant a whole new repo? More so if issues/PRs are being dropped.  


You could consider [just renaming](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) this repo to `volumio-core` or what ever.. You keep all your old issues, PRs, forks, and also all the publicity from stars,followers,achievements etc all in one place.
Github will automatically forward links form github.com/volumio/Volumio2 to this new name. 

You also solve the issue of people searching for v2 support/code as `master` would point to well, current code, and v2 could be kept as a `jessie` snapshot.  

> If you can help by filtering the ones that are more relevant and port them, that would be appreciated

Personally, I am not a fan of issue management at $dayjob, let alone in my free time - so not going to be much help. I did close all my open issues though.
